src/successModal/SuccessModal.tsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react'
 import uuidv4 from 'uuid/v4'
 
-import { ModalProps } from 'modal'
 import { ButtonStatus } from 'button'
 
+import { ModalProps } from '../modal'
```

Comment:
Can you add a comment on the issue and why this is working?
Also would it be possible that other components have the same issue, but we just don't know yet because SPA is not fully migrated in TS?

Reply:
the alias is not being correctly transformed on _build_, so yes! this is a potential problem for SPA and its migration on TS. More we migrate files, more we are going to find this kind of issues. :)

Reply:
If the problem is that kairos is not able to resolve the `'modal'` path (which is normal because it has no way of knowing where to find it), the issue will appear for every type that reference at least one from another file.

The compiled JS works fine because the path are relative:

```js
// build/successModal/SuccessModal.js

// ...
var _button = require("../button");
var _display = _interopRequireDefault(require("../typography/display1"));
// ...
```

Reply:
Is it also the cause of all these warnings when we `npm start`?

```
WARNING in ./src/topBar/index.tsx 17:0-39
"export 'TopBarProps' was not found in './TopBar'
 @ ./src/pages/login/email.story.tsx
 @ ./stories/index.tsx
 @ ./stories sync ^\.\/.*$
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true

WARNING in ./src/tripCard/index.tsx 17:0-49
"export 'TripCardProps' was not found in './TripCard'
 @ ./src/layout/section/cardsSection/story.tsx
 @ ./stories/index.tsx
 @ ./stories sync ^\.\/.*$
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true

WARNING in ./src/uneditableTextField/index.tsx 18:0-65
"export 'UneditableTextFieldProps' was not found in './UneditableTextField'
 @ ./src/uneditableTextField/story.tsx
 @ ./stories/index.tsx
 @ ./stories sync ^\.\/.*$
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true

WARNING in ./src/tripCard/index.tsx 17:0-49
"export 'User' was not found in './TripCard'
 @ ./src/layout/section/cardsSection/story.tsx
 @ ./stories/index.tsx
 @ ./stories sync ^\.\/.*$
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true

WARNING in ./src/why/index.tsx 17:0-33
"export 'WhyProps' was not found in './Why'
 @ ./src/why/story.tsx
 @ ./stories/index.tsx
 @ ./stories sync ^\.\/.*$
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true&quiet=true
```

Reply:
Not sure 🤔